OrbitCore/Callstack.h

```diff
@@ -12,15 +12,15 @@
 
 #include "CallstackTypes.h"
 
-struct CallStack {
-  CallStack() = default;
+struct HashedCallstack {
+  HashedCallstack() = default;
   inline CallstackID Hash() {
-    if (m_Hash != 0) return m_Hash;
-    m_Hash = XXH64(m_Data.data(), m_Data.size() * sizeof(uint64_t), 0xca1157ac);
-    return m_Hash;
+    if (hash != 0) return hash;
```

Comment:
and calculate this hash in c-tor, which will let us make this method const.

(please ignore this if we are planning to get rid of this struct in the nearish future)